### ISSUE TITLE
Wrong webdomain results in CORS issue

### ISSUE THREAD

**Issue by philippe-gregoire** [IBM-Cloud/openwhisk-cognos-dashboard#1]
I think that the webDomain = 'https://service.us.apiconnect.ibmcloud.com' will cause this module to fail when the function is exposed with another web domain (e.g. https://123abcdef.us-south.apiconnect.appdomain.cloud/dashboard/sample-view)
This is at line 55 in `/src/Dashboard.ts`

**Comment by van-ibm** [comment]
Been a while since I looked at the code, but the webDomain is parameterized. So you should bind the webDomain parameter hosting the function during the deployment (see [Working with parameters](https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-working-with-parameters#default-params-package)).  This *should* override the default value `https://service.us.apiconnect.ibmcloud.com`.

**Comment by philippe-gregoire** [comment]
OK, makes sense. Would be worth detailing in the readme?

**Comment by van-ibm** [comment]
Yes.  I went back and reviewed the README and was surprised I didn't have a step.  But at the time I mimicked the format of other packages where it simply listed parameters.  I'll try to make it more clear in the initial setup within the README.

**Comment by philippe-gregoire** [comment]
Thanks!
[In any case, your sample helped me figure out a mistake I was making in another test, and discover the use of functions with CDE, thanks again!]
